app/templates/briefs/_base_edit_question_page.html

```diff
@@ -16,8 +16,7 @@
 {% block mainContent %}
 
   <!-- Pricing questions use a prefix, which is available in govuk-frontend v3, but not in v2. -->
```

Comment:
```suggestion
  {# Pricing questions use a prefix, which is available in govuk-frontend v3, but not in v2. #}
```
can we use Jinja style comments so they don't leak into the HTML?